api-reference/50 Common/Object Structures/globalConfig/defaultUseCurrencyAccountingStyle.md

```diff
@@ -5,7 +5,18 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to apply the accounting style to formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The accounting style adds parentheses to negative numbers instead of the minus sign. To disable the accounting style globally and display formatted numbers of the `currency` [type](/Documentation/ApiReference/Common/Object_Structures/format/#type) with the minus sign, set this property to **false**.
```

Comment:
This topic does not include a connection to the corresponding "local" accounting style flag that corresponds to individual components and has higher priority than the currently described setting. I don't believe that a See Also link is enough to cover that.